docs/source/how_to_guides/configure_cloud_storage_credentials.md

```diff
@@ -206,3 +207,29 @@ export AZURE_ACCOUNT_NAME='test'
 export AZURE_ACCOUNT_ACCESS_KEY='NN1KHxKKkj20ZO92EMiDQjx3wp2kZG4UUvfAGlgGWRn6sPRmGY/TEST/Dri+ExAmPlEExAmPlExA+ExAmPlExA=='
 ```
 ````
+
+## Databricks File System (DBFS)
```

Comment:
Looks like the Doc CI is failing. Can you run the below command and see if passes for you locally?

```
cd docs
pip install -e '.[docs]'
make clean && make html
make host   # open the output link in a browser.
```

Reply:
I tried make clean && make html and it said 
Notebook error:
PandocMissing in examples/cifar10.ipynb:
Pandoc wasn't found.
Please check that pandoc is installed:
https://pandoc.org/installing.html
make: *** [html] Error 2